packages/phone-number-privacy/signer/src/common/error-utils.ts

```diff
@@ -9,8 +9,6 @@ export enum ErrorMessages {
   INVALID_INPUT = 'CELO_PNP_ERR_04 Invalid input paramaters',
   EXCEEDED_QUOTA = 'CELO_PNP_ERR_05 Requester exceeded salt service query quota',
   SIGNATURE_COMPUTATION_FAILURE = 'CELO_PNP_ERR_06 Failed to compute BLS signature',
-  DUPLICATE_REQUEST_TO_MATCHMAKE = 'CELO_PNP_ERR_08 Attempt to request >1 matchmaking',
-  UNVERIFIED_USER_ATTEMPT_TO_MATCHMAKE = 'CELO_PNP_ERR_09 Unverified user attempting to matchmake',
```

Comment:
do you no think this should just be consolidated to common folder?  removing #8 and #9 is a little confusing otherwise.  and in the future, we discussed that signer would manage matchmaking

Reply:
Yeah that's fair, I'll revert this